tasks/AccelerationController.cpp

```diff
@@ -23,12 +23,14 @@ AccelerationController::~AccelerationController()
 
 bool AccelerationController::configureHook()
 {
-    if (! AccelerationControllerBase::configureHook()){
+    if (!AccelerationControllerBase::configureHook())
+    {
         return false;
     }
-    // Warning: By default, the storage orders in Eigen is column-major. So the thrusterMatrix will be gotten transposed.
-    thrusterMatrix = _matrix.get();
-    unsigned int numberOfThrusters = thrusterMatrix.cols();
+
+    // Warning: By default, the storage order in Eigen is column-major. So a matrix is gotten transposed when loaded from a YAML file. That's why the usage of the transpose method at the thrusterMatrix is needed.
```

Comment:
// Warning: By default, the storage order in Eigen is column-major.
// This means that it is in effect transposed on load when compared
// with how it looks in YAML, that is the YAML
//
// thrusterMatrix:
//    data: [1, 2, 3,
//               4, 5, 6]
//
// Will actually be loaded in thrusterMatrix as row(0) = [1, 4], row(1) = [2, 5] ...
//
// The transpose here makes sure that one that writes YAML config files
// gets what (s)he expects.

Reply:
Ping @seixasxbr